lib/scraped/response/decorator/absolute_urls.rb

```diff
@@ -7,23 +7,31 @@ class Decorator
       class AbsoluteUrls < Decorator
         def body
           Nokogiri::HTML(super).tap do |doc|
+            @doc = doc
```

Comment:
As a general rule of thumb, you should try to avoid setting up new instance variables anywhere other than a constructor (or used for memoisation, and only accessed from within the same method).